packages/fxa-profile-server/lib/img-workers.js

```diff
@@ -15,22 +15,47 @@ const WORKER_URL = config.get('worker.url');
 const ACCEPT_ENCODING_HEADER = 'identity';
 
 exports.upload = function upload(id, payload, headers) {
-  return new P(function (resolve, reject) {
+  return new P(function(resolve, reject) {
     var url = WORKER_URL + '/a/' + hex(id);
 
+    var header_excludes = config.get('worker.headers_exclude');
+    var header_includes = config.get('worker.headers_include');
```

Comment:
Let's use `const` instead of `var` here.

Reply:
done